core/src/main/java/org/elasticsearch/index/query/FuzzyQueryParser.java

```diff
@@ -60,23 +68,23 @@ public FuzzyQueryBuilder fromXContent(QueryParseContext parseContext) throws IOE
                 if (token == XContentParser.Token.FIELD_NAME) {
                     currentFieldName = parser.currentName();
                 } else {
-                    if ("term".equals(currentFieldName)) {
+                    if (parseContext.parseFieldMatcher().match(currentFieldName, TERM_FIELD)) {
```

Comment:
After going throught the whole PR I'd probably not suggest this anymore, unless you have some nifty awk / ide search&replace magic at hand. Sorry for the noise.